src/Polly.Shared/PolicyResult.cs

```diff
@@ -117,7 +143,13 @@ internal static PolicyResult<TResult> Failure(Exception exception, ExceptionType
                 context);
         }
 
-        internal static PolicyResult<TResult> Failure(TResult handledResult, Context context)
+        /// <summary>
+        /// Builds a failed <see cref="PolicyResult" />
+        /// </summary>
+        /// <param name="handledResult">The handled result</param>
+        /// <param name="context">The policy context</param>
+        /// <returns></returns>
```

Comment:
((suggest add, for 'returns'))

`A <see cref="PolicyResult" /> representing a failed execution through the policy.`